src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -205,9 +207,9 @@ func doMain() error {
 		log.Debug("Running proxy")
 		if cfg.IsDownstream() {
 			// This will open a proxy on the address and port given by -addr
-			go runClientProxy(cfg)
+			runClientProxy(cfg)
```

Comment:
No need to spawn goroutine here as the code block is already wrapped in a goroutine.